src/test/java/org/apache/commons/lang3/time/DurationUtilsTest.java

```diff
@@ -38,16 +38,16 @@ public class DurationUtilsTest extends AbstractLangTest {
 
     @Test
     public void testGetNanosOfMilli() {
-        assertEquals(0, DurationUtils.getNanosOfMiili(null));
-        assertEquals(0, DurationUtils.getNanosOfMiili(Duration.ZERO));
-        assertEquals(1, DurationUtils.getNanosOfMiili(Duration.ofNanos(1)));
-        assertEquals(10, DurationUtils.getNanosOfMiili(Duration.ofNanos(10)));
-        assertEquals(100, DurationUtils.getNanosOfMiili(Duration.ofNanos(100)));
-        assertEquals(1_000, DurationUtils.getNanosOfMiili(Duration.ofNanos(1_000)));
-        assertEquals(10_000, DurationUtils.getNanosOfMiili(Duration.ofNanos(10_000)));
-        assertEquals(100_000, DurationUtils.getNanosOfMiili(Duration.ofNanos(100_000)));
-        assertEquals(0, DurationUtils.getNanosOfMiili(Duration.ofNanos(1_000_000)));
-        assertEquals(1, DurationUtils.getNanosOfMiili(Duration.ofNanos(1_000_001)));
+        assertEquals(0, DurationUtils.getNanosOfMilli(null));
```

Comment:
Hi @arturobernalg 
Thank you for your PR.
We have to test both calls explicitly, so your best bet is to clone the test method.

Reply:
OK, done. Changed.
TY @garydgregory